noir-projects/noir-contracts/contracts/token_with_refunds_contract/src/main.nr

```diff
@@ -469,35 +469,40 @@ contract TokenWithRefunds {
         // to the user in the `complete_refund(...)` function.
         storage.balances.sub(user, U128::from_integer(funded_amount)).emit(encode_and_encrypt_note_with_keys(&mut context, user_ovpk, user_ivpk, user));
 
-        // 4. We create the partial notes for the fee payer and the user.
+        // 4. Now we "manually" compute the slots (by setting the slots we insert the notes to the balances map under a correct key)
+        let fee_payer_balances_slot = derive_storage_slot_in_map(TokenWithRefunds::storage().balances.slot, fee_payer);
+        let user_balances_slot = derive_storage_slot_in_map(TokenWithRefunds::storage().balances.slot, user);
+
+        // 5. We create the partial notes for the fee payer and the user.
         // --> Called "partial" because they don't have the amount set yet (that will be done in `complete_refund(...)`).
         let fee_payer_partial_note = TokenNote {
-            header: NoteHeader::empty(),
+            header: NoteHeader {
+                contract_address: AztecAddress::zero(),
```

Comment:
I am just setting the address to zero since it's unused.

@nventuro would make the decision whether the nuke it as it's looking pretty weird here.